charts/backstage/values.yaml

```diff
@@ -86,9 +86,9 @@ ingress:
 
 backstage:
   image:
-    registry: ""
-    repository: ""
-    tag: ""
+    registry: ghcr.io
+    repository: backstage/backstage
+    tag: latest
```

Comment:
don't forget to keep the [documentation](https://github.com/backstage/charts#backstage-parameters) in sync with these changes, until we have a better way to automatically sync those 😅

Also (out of the scope of this PR) [here](https://github.com/backstage/charts#prerequisites) we are specifying that the users need to provide their own Backstage Docker image. Perhaps we could specify that the chart uses a default official Docker image but we encourage the users to build and provide their own? WDYT?

Reply:
You are right. I've updated the docs, WDYT?

Reply:
fantastic!